Documentation/gettingstarted/hubble-configuration.rst

```diff
@@ -196,3 +196,42 @@ resolve this issue. Pick one of the options below:
                     --set webhook.tolerations='["operator": "Exists"]'
 
         Then configure an issuer and install Cilium.
+
+.. _hubble_configure_metrics_tls:
+
+Metrics TLS and Authentication
+===============================
+
+Starting with Cilium 1.16, Hubble supports configuring TLS on the Hubble metrics API in addition to the Hubble observer API.
+
+This can be done by specifying the listed options below to Helm at install or upgrade time, along with your existing Hubble TLS configuration options mentioned above.
```

Comment:
Translators sometimes struggle with this kind of usage of "below" and "above". Can we reword this section?

```suggestion
This can be done by specifying the following options to Helm at install or upgrade time, along with the TLS configuration options described in the previous section
```